sys-power/phc-intel/phc-intel-0.3.2_p45.ebuild

```
@@ -0,0 +1,53 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-mod-r1
+
+DESCRIPTION="Frequency driver for Intel CPUs with undervolting feature"
+HOMEPAGE="https://gitlab.com/linux-phc/phc-intel"
+REV="rev${PV#*_p}"
+SRC_URI="https://gitlab.com/linux-phc/phc-intel/-/archive/${REV}/${PN}-${REV}.tgz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+S="${WORKDIR}/${PN}-${REV}"
+RESTRICT="strip test"
+
+CONFIG_CHECK="
+	CPU_FREQ
+	ACPI_PROCESSOR
+"
+
+pkg_setup() {
+	linux-mod-r1_pkg_setup
+
+	if linux_chkconfig_builtin X86_ACPI_CPUFREQ; then
+		eend 1
+		eerror "Kernel driver acpu_freq is compiled into the kernel."
+		eerror "Unlike modules, compiled-in drivers cannot be replaced."
+		eerror "Please set CONFIG_CPU_X86_ACPI_CPUFREQ=m or n."
+		die "Incorrect kernel configuration options"
+	fi
+}
+
+src_compile() {
+	local modlist=( phc-intel )
+	local modargs=( KSRC=${KV_DIR} KBUILD=${KV_OUT_DIR} )
+
+	linux-mod-r1_src_compile
+}
+
+src_install() {
+	linux-mod-r1_src_install
+
+	echo 'blacklist acpi_cpufreq' > "${T}/${PN}".conf
```

_if_ we were keeping the `echo`..
```suggestion
	echo 'blacklist acpi_cpufreq' > "${T}/${PN}".conf || die
```

But we don't need `echo` here:
```
newins - ${PN}.conf <<-EOF
blacklist ...
EOF
```

---

Done. Thank you for the quick review!